curse.go

```diff
@@ -40,7 +40,7 @@ func New() (*Cursor, error) {
 	c := &Cursor{}
 	c.Position.X, c.StartingPosition.X = col, col
 	c.Position.Y, c.StartingPosition.Y = line, line
-	c.terminal, err = term.New()
+	c.terminal, err = term.Open("/bin/stty")
```

Comment:
Hm, weird, I can take a look after the weekend, if you not figure this out by then that is